hnn_core/network.py

```diff
@@ -1340,6 +1415,310 @@ def plot_cells(self, ax=None, show=True):
         """
         return plot_cells(net=self, ax=ax, show=show)
 
+    def write(self, fname, overwrite=True, save_unsimulated=False):
+        # Tasks
+        # 1) Remove params dict (Partial done)
+        # 2) Add overwrite warnings (Done)
+        # 3) Add boolean to read/write un-simulated network (Done)
+        # 4) Add boolean to read/write simulated network (Done)
+        # 5) Think about documentation
+        # 6) Add tests
+        # 7) Store the type of object saved and issue warning (Done)
```

Comment:
suggest moving this to the PR description ... github has a nice checkbox feature you can use

Reply:
Ooh its a pretty cool feature. I just saw it.